pom.xml

```diff
@@ -104,12 +106,15 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-jar-plugin</artifactId>
           <version>3.2.0</version>
-        </plugin>
-        <plugin>
-          <groupId>org.apache.felix</groupId>
-          <artifactId>maven-bundle-plugin</artifactId>
-          <version>5.1.2</version>
-          <inherited>true</inherited>
```

Comment:
Have removed this because it looks like this is not used (unless I am overlooking something). The `gson` Maven submodule is using the `bnd-maven-plugin`.